### PR TITLE
Improve YUIDoc comments and fix build warnings

### PR DESCRIPTION
**Pull request by gilest**
Was seeing a few YUIDoc warnings in my console.

Added missing `@method` tags in `WithFilesMixin` and `Queue`.

Also removed a comment documenting the `init()` method in `FileQueueService`. My reasoning for this is.

- `@constructor` tag is meant to be used in a class block only - it's invalid YUIDoc
- If I tag this code block with `@method` then the `init()` method will show up on the docs site
- In my opinion this comment is superfluous

### CODE REVIEW

**Comment by jelhan** [comment]
Ups, this should have been part of `3.0.0` isn't it? Guess we just have a patch release soon.

**Comment by gilest** [comment]
It's quite a subtle change so not important to cut a new release IMO

**Comment by jelhan** [comment]
> It's quite a subtle change so not important to cut a new release IMO

But on the other hand it's not much work thanks to the automation. :smile:

And at least for me doing small releases is even less work as you don't have to worry to miss something important. If you are sure what will be bundled in it's just a serious of `release-it`, hitting enter a few times and entering OTP for NPM.

TL;DR: Released v3.0.1. :tada:
